sharing-service/services/sharing/lib/getSSMValue.js

```diff
@@ -0,0 +1,18 @@
+const AWS = require('aws-sdk')
+const SSM = new AWS.SSM()
+const log = require('slic-tools/log')
+async function getSSMValue() {
```

Comment:
The fact that this function and the module it contains have the same name seems odd. What is the purpose of the module? Name it accordingly. Within the module, what is the purpose of the function? The name `getSSMValue` does not describe which value it gets, it only describes how it is implemented (using SSM!)